src/_preflights/tw-reset.js

```diff
@@ -82,10 +84,15 @@ Reset links to optimize for opt-in styling instead of opt-out.
 */
 
 a {
-  color: inherit;
+  color: var(--w-color-text-link);
   text-decoration: inherit;
 }
```

Comment:
Should we also apply `cursor-pointer` and `no-underline` classes here as in[ Fabric's base.css](https://github.com/fabric-ds/css/blob/01fd80a27461cf0d593911e974853f84963d0681/src/base.css#L49-L50)?